Feat: update faq when upgrade namespace of Model class used (#67)
dunglas/doctrine-json-odm: README.md

As a side note: If you happen to use [Autowiring](https://symfony.com/doc/current/service_container/autowiring.html) in your `services.yaml` you might need to set `autowire: false` too.

> **When namespace of Model used change**

Reviewer: ```suggestion
**When the namespace of an entity used changes**
```

> You must have migrated the data in the database because your JSON has `#type` key which contains the namespace of your used Model class.

Reviewer: ```suggestion
Because we store the `#type` along with the data in the database, you have to migrate the already existing data in your database to reflect the new namespace.
```

> For example if we have a project that we migrate to Symfony/flex,

Reviewer: What do you think about this (it's not related to Symfony Flex at all so I left that away):

```
Example: If we have a project that we migrate from `AppBundle` to `App`, we have the namespace `AppBundle/Entity/Bar` in our database which has to become `App/Entity/Bar` instead.
```

Author: it's much better, thank you ;)

we will have as namespace `AppBunle/Entity/Bar` for our Model class used who will become `App/Entity/Bar`.

> With MySQL we will have to do this:

Reviewer: ```suggestion
When you use `MySQL`, you can use this query to migrate the data:
```

```

> UPDATE Baz SET misc = JSON_REPLACE(misc, '$."#type"', 'App\\\Entity\\\Bar');

Reviewer: I don't like this example. It will replace everything, not just what was `AppBundle\Entity\Bar`. So this is not the correct query.
